samples/install-without-bom/pom.xml

```diff
@@ -22,6 +22,8 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <arrow.version>15.0.2</arrow.version>
+    <opentelemetry-exporter-logging.version>1.38.0</opentelemetry-exporter-logging.version>
```

Comment:
These can move directly into the dep as it's only referenced once.

Reply:
done